pubnub-chat-impl/src/jsMain/kotlin/ChatJs.kt

```diff
@@ -328,6 +328,7 @@ class ChatJs internal constructor(val chat: ChatInternal, val config: ChatConfig
                 createJsObject<GetUnreadMessagesCountsJs> {
                     this.channel = unreadMessagesCount.channel.asJs(this@ChatJs)
                     this.membership = unreadMessagesCount.membership.asJs(this@ChatJs)
+                    this.page = MetadataPage(unreadMessagesCount.next, unreadMessagesCount.prev)
```

Comment:
Maybe we could add test for paging in JS. 
There is already test for getUnreadMessagesCounts called : test("Should get the number of unread messages on channels"

located:
kmp-chat/js-chat/tests/channel.test.ts